excel/src/main/java/org/apache/metamodel/excel/ExcelUtils.java

```diff
@@ -153,20 +155,15 @@ public static Workbook readWorkbook(ExcelDataContext dataContext) {
      * @param wb
      */
     public static void writeAndCloseWorkbook(ExcelDataContext dataContext, final Workbook wb) {
-        // first write to a temp file to avoid that workbook source is the same
-        // as the target (will cause read+write cyclic overflow)
```

Comment:
I think the gist of this is just that you're disregarding this comment now? That would cause an old bug to reoccur.